src/FsAutoComplete/LspHelpers.fs

```diff
@@ -759,8 +777,11 @@ type FSharpConfig =
     InterfaceStubGenerationMethodBody: string
     AddPrivateAccessModifier: bool
     UnusedOpensAnalyzer: bool
+    UnusedOpensAnalyzerExclusions: Regex array
```

Comment:
Since we already have `FSharp.unusedOpensAnalyzer` it would be a breaking change to make it a nested object and all client configs would need to update for this. Could be renamed to whatever makes more sense though.